modules/ROOT/pages/endpoint-https-receive.adoc

```diff
@@ -40,28 +40,59 @@ Possible values include:
+
+=== Deploy to CloudHub on Private Port
+
+When an HTTP or HTTPS receive endpoint is deployed with the private port configuration enabled, the endpoint is deployed to the private HTTPS port (8092) and the internal URL of the endpoint appears on the the *Message Flow* or *Endpoint* detail page.
+
+After deploying the first message flow using the HTTP or HTTPS receive endpoint configuration, you can view the runtime application name for the endpoint from the *Endpoint* details page or from the *Receive* section of your message flow. To configure xref:runtime-manager/lb-mapping-rules[URL mapping rules] in the xref:runtime-manager::cloudhub-dedicated-load-balancer.adoc[Dedicated Load Balancer] to forward HTTP or HTTPS messages received from your backend applications to outbound B2B message flows, provide the application name to your organization’s platform administrator.
+
+If your backend processes consuming the HTTP or HTTPS receive endpoints are also running inside the same VPC, they can call the internal URL directly.
+
+If your backend processes consuming the endpoint are in a different network, your organization’s platform administrator can update Dedicated Load Balancer settings to xref:runtime-manager::lb-whitelists.adoc[whitelist] specific IP addresses to access the endpoint.
+
+The external URL that you will share with your partners will be https://{DLB-domain}/{Input-path}/{base-path}/{message-type}.
+
+For example, if the name of your DLB domain is `mythical.lb.anypointdns.net`, and the runtime application name is `b2b-inbound-http-ognq`, and the base path you configured in the HTTPS receive endpoint is `send2`, then your administrator will add a URL mapping rule in the Dedicated Load Balancer settings to forward incoming requests to the runtime application within your VPC.
+
+image::URL-mapping-rules-http.png[URL mapping rules in the Dedicated Load Balancer]
+
+With these configurations:
+* The external HTTPS receive endpoint URL is https://mythical.lb.anypointdns.net/apm/qa/outbound/send2/.
+* Your backend applications will be sending the application JSON/XML messages for your outbound message flows from https://mythical.lb.anypointdns.net/apm/qa/outbound/send2/{message-type}.
+* If `enterprise-ob-po-ack` is the message type, your backend systems post the HTTPS requests to the URL
+https://mythical.lb.anypointdns.net/apm/qa/outbound/send2/enterprise-ob-po-ack.
+
+=== Deploy to On-Premise Mule Runtime
+
+If you are deploying to an on-premises Mule instance with a firewall and load balancer, work with your hosting or network security team to obtain the externally accessible DNS or name of the host where the Mule server is running, and replace `{runtime_host}` in the URL with the DNS name of your host. For example, if your DNS host name is `b2b.mycompany.com`, the receiver URL is `+https://b2b.mycompany.com/base-path/+`, and your backend systems should be posting the HTTPS request to https://b2b.mycompany.com/base-path/{message-type}.
```

Comment:
Mule server? Do you mean Mule runtime engine (Mule)?

Reply:
Also, "For example, if your DNS host name is `b2b.mycompany.com`, the receiver URL is `+https://b2b.mycompany.com/base-path/+` and your backend systems post the HTTPS request to `https://b2b.mycompany.com/base-path/{message-type}`.
```

Reply:
Done